### PR TITLE
1st commit

### PR DESCRIPTION
**Pull request by stevenchung213**
test run after workflow reset

### CODE REVIEW

File: client/components/overview.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <div>hello world</div>
+      </React.Fragment>
+    );
+  }
+}
```
Comment (mtiosejo) [inline client/components/overview.jsx]: Delete new line at end of this file.

File: db/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  area: Number,
+  bedrooms: Number,
+  sleeps: Number,
+  bathrooms: Number,
+  halfBaths: Number,
+  minStay: Number,
+  description: String
+});
+
+let overview = mongoose.model('overview', overviewSchema);
```
Comment (mtiosejo) [inline db/index.js]: Change 'let' to 'const' since you'll probably not change your DB model

File: db/seed.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+const db = require('./index.js');
+
+const range = function (start, stop, step) {
```
Comment (mtiosejo) [inline db/seed.js]: The function declaration needs changed to an ES6 arrow function if when using 'const'

File: db/seed.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+const db = require('./index.js');
+
+const range = function (start, stop, step) {
+  step = step || 1;
+  let result = [];
+  for (let i = start; i <= stop; i += step) {
```
Comment (mtiosejo) [inline db/seed.js]: Re-implement for loop to a HOF method to adhere to AirBnB style guide

File: db/seed.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+const range = function (start, stop, step) {
+  step = step || 1;
+  let result = [];
+  for (let i = start; i <= stop; i += step) {
+    result.push(i);
+  }
+  return result;
+};
+
+const generator = function (amount) {
```
Comment (mtiosejo) [inline db/seed.js]: The function declaration needs changed to an ES6 arrow function if when using 'const'

File: db/seed.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+  amount = amount || 1;
+
+  let docsArray = [];
+
+  let types = ['House', 'Condo', 'Apartment', 'Townhouse', 'Cabin'];
```
Comment (mtiosejo) [inline db/seed.js]: use a more specific variable name than 'types'

File: db/seed.js [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  ' senectus et netus et malesuada fames. Sollicitudin tempor id eu nisl. Turpis cursus in hac habitasse platea. Morbi tristique' +
+  ' senectus et netus et malesuada fames ac. Quisque sagittis purus sit amet volutpat consequat mauris nunc. Est placerat in egestas' +
+  ' erat imperdiet sed euismod nisi.', 'Viverra aliquet eget sit amet tellus cras adipiscing enim. Fames ac turpis egestas maecenas' +
+  ' pharetra convallis. Quam viverra orci sagittis eu volutpat odio. Dolor purus non enim praesent elementum facilisis. Dignissim cras' +
+  ' tincidunt lobortis feugiat vivamus. Senectus et netus et malesuada fames ac turpis.', 'Consectetur purus ut faucibus pulvinar' +
+  ' elementum integer. Risus ultricies tristique nulla aliquet enim tortor. Sodales neque sodales ut etiam sit. Elit ut aliquam purus' +
+  ' sit amet luctus venenatis lectus. Sapien pellentesque habitant morbi tristique senectus et. Mauris cursus mattis molestie a iaculis' +
+  ' at erat pellentesque. In nisl nisi scelerisque eu ultrices vitae auctor eu augue. Odio tempor orci dapibus ultrices in iaculis nunc.']; // length of 5
+
+  let random = function (array) {
```
Comment (mtiosejo) [inline db/seed.js]: The function declaration needs changed to an ES6 arrow function if when using 'const'

File: db/seed.js [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    return Math.floor(Math.random() * array.length);
+  };
+
+  for (let i = 0; i < amount; i++) {
```
Comment (mtiosejo) [inline db/seed.js]: Re-implement for loop to a HOF method to adhere to AirBnB style guide

File: db/seed.js (hunk 7 above)
Comment (mtiosejo) [inline db/seed.js]: Change 'let' to 'const' for functions

File: db/seed.js [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+db.overview.create(documents);
+
+
+/*
+  let area1br = range(1000, 1600, 50);
+  let area2br = range(1200, 1900, 50);
+  let area3br = range(1700, 2400, 50);
+  let area4br = range(2200, 3000, 50);
+  let area5br = range(2700, 4000, 50);
+*/
```
Comment (mtiosejo) [inline db/seed.js]: Create new line at end of this file.

File: public/index.html [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+	<meta charset="UTF-8">
+	<title>Overview</title>
+</head>
+<body>
+<div id="overview"></div>
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+<script src="app.js"></script>
+</body>
+</html>
```
Comment (mtiosejo) [inline public/index.html]: Create new line at end of this file.

File: public/index.html [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (mtiosejo) [inline public/index.html]: Need to correct this HTML file for proper 2-spaced indentation

File: server/index.js [hunk 12]
```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+let app = express();
```
Comment (mtiosejo) [inline server/index.js]: Change 'let' to 'const'

File: server/index.js [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+
+
+
+
+let port = 3000;
+
+app.listen(port, function() {
+  console.log(`listening on port ${port}`);
+});
```
Comment (mtiosejo) [inline server/index.js]: Create new line at end of this file.

File: server/index.js [hunk 14]
```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+let app = express();
+
```
Comment (mtiosejo) [inline server/index.js]: Delete unnecessary white spaces in this document

File: test/fec.test.js [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+    expect(true).toBe(true);
+  });
+
+  test('Make sure Enzyme method mount is working', () => {
+    const divTest = <div>hello world</div>;
+    const expected = mount(divTest);
+    expect(expected.text()).toBe('hello world');
+  });
+
+});
```
Comment (mtiosejo) [inline test/fec.test.js]: Create new line at end of this file.

File: test/fec.test.js [hunk 16]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+
+describe('Jest/Enzyme tests', () => {
+
```
Comment (mtiosejo) [inline test/fec.test.js]: Delete unnecessary white spaces

File: test/tests.js [hunk 17]
```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```
Comment (mtiosejo) [inline test/tests.js]: Create new line at end of this file.

File: webpack.config.js [hunk 18]
```diff
@@ -0,0 +1,21 @@
...
+          }
+        }
+      }
+    ]
+  },
+  output: {
+    filename: 'app.js',
+    path: __dirname + '/public'
+  }
+};
```
Comment (mtiosejo) [inline webpack.config.js]: Create new line at end of this file.

File: db/seed.js [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+    docsArray.push(document);
+  }
+  return docsArray;
+};
+
+let documents = generator(500)/*?*/;
+
+db.overview.create(documents);
+
+
```
Comment (mtiosejo) [inline db/seed.js]: Delete unnecessary white space and unused comments

File: db/seed.js [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+      minStay: minStay[random(minStay)],
+      description: propDesc[random(propDesc)]
+    };
+
+    docsArray.push(document);
+  }
+  return docsArray;
+};
+
+let documents = generator(500)/*?*/;
```
Comment (mtiosejo) [inline db/seed.js]: Consider using const, unless you plan to re-assign a value to 'documents'

File: client/index.jsx [hunk 21]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Overview from './components/overview.jsx';
+
+
```
Comment (mtiosejo) [inline client/index.jsx]: delete unnecessary white spaces

**Comment by stevenchung213** [comment]
all changes accounted for

**Pull request closed without merging**